File: JavaScript/Inserción_al_final.js
let arreglo = [10, 20, 30, 0, 0, 0, 0, 0, 0, 0];
let n = 3;

function insFinal(valor) {
    arreglo[n] = valor;
    n++;
}

insFinal(40);
console.log(arreglo.slice(0, n));

File: Python/Inserción_al_final.py
arreglo = [10, 20, 30, 0, 0, 0, 0, 0, 0, 0]
n = 3

def ins_final(valor):
    global n
    arreglo[n] = valor
    n += 1

ins_final(40)
print(arreglo[:n])
